src/tests/Interop/PInvoke/Array/MarshalArrayAsParam/AsDefault/AsDefaultTest.cs

```diff
@@ -395,7 +395,7 @@ private static void TestMarshalInOut_ByVal()
         Console.WriteLine("CStyle_Array_Int_InOut_ZeroLength");
         int[] iArrLength0 = InitArray<int>(0);
         Assert.True(CStyle_Array_Int_InOut_ZeroLength(iArrLength0));
-        Assert.Equal(0, iArrLength0.Length);
+        Assert.Empty(iArrLength0);
```

Comment:
similarly, I have no idea whether int equality or `Empty` is more dangerous in the various configurations

Reply:
I just fixed those as the analyzer slapped me because of them and it seemed to me that somehow exempting them from the scanning would be more complicated than just going with the flow.